webapp/app/__main__.py

```diff
@@ -1,31 +1,61 @@
-import webbrowser
-
 import streamlit as st
+# import webbrowser
 
 st.title("Welcome to Cat Kibble Box -CKB")
 
 status = st.header("What would you like to do? ")
-hobby = st.selectbox("Kibble-Box: ", ["Box001", "Box002", "Box003"])
+kibblepi = st.selectbox("Kibble-Box: ", ["Box001", "Box002", "Box003"])
 
-st.write("Your Kibble-box  is: ", hobby)
+st.write("Your Kibble-box  is: ", kibblepi)
 
 options = st.multiselect("Options available: ", ["Camera", "Email", "Eat"])
 
-st.write("You selected", len(hobby), "options")
+st.write("You selected", len(options), "options")
 
 
-if st.button("Submit"):
-    webbrowser.open("https://www.marca.com")
-    st.success("Successful operation!!!!!")
+# if st.button("Submit"):
+#     webbrowser.open("https://www.marca.com")
+#     st.success("Successful operation!!!!!")
 
 
 st.write(
     f"""
-    <a target="_self" href="https://marca.com">
+    <a target="_self" href="http://devrulls.ddns.net:8080/">
+        <button>
+            RaspberryPi - Home
+        </button>
+    </a>
+    """,
+    unsafe_allow_html=True,
+)
+st.write(
+    f"""
+    <a target="_self" href="http://devrulls.ddns.net:8080/send-email">
+        <button>
+            Send Email
+        </button>
+    </a>
+    """,
+    unsafe_allow_html=True,
+)
+st.write(
+    f"""
+    <a target="_self" href="http://devrulls.ddns.net:8080/push-button">
         <button>
-            redirect via Marca
+            Push-Button
         </button>
     </a>
     """,
     unsafe_allow_html=True,
 )
+st.write(
+    f"""
+    <a target="_self" href="http://devrulls.ddns.net:8080/servo-kibblepi">
+        <button>
+            Servo-Motor
+        </button>
+    </a>
+    """,
+    unsafe_allow_html=True,
+)
+
```

Comment:
We should find a way to clean this file as well. We'll do that together ;)

Reply:
Could we review it together?